src/tests/ttls/enc.cpp

```diff
@@ -49,16 +49,16 @@ void test_ecall()
         "-----END CERTIFICATE-----\\r\\n";
 
     constexpr std::string_view kRequest = "GET / HTTP/1.0\r\n\r\n";
-    const auto ttls_cfg =
-        "{\"tls\":{\"localhost:9000\": \" " + kCACrt + " \" }}";
+    const auto ttls_cfg = "{\"tls\":{\"localhost:9000\": {\"cacrt\": \" " +
+                          kCACrt + " \", \"clicert\":\"\", \"clikey\":\"\" }}}";
     OE_TEST(oe_load_module_host_socket_interface() == OE_OK);
 
     const int fd = socket(AF_INET, SOCK_STREAM, 0);
     OE_TEST(fd != -1);
 
     ert_init_ttls(ttls_cfg.data());
 
-    auto t1 = edgeless::ttls::StartTestServer();
+    auto t1 = edgeless::ttls::StartTestServer(0);
```

Comment:
I didn't want to include mbedtls headers just to use MBEDTLS_SSL_VERIFY_NONE instead of 0. Either I include the header or I can explain the options in the `StartTestServer()` comment in a future ttls update. 
What do you think?

Reply:
I agree not to include the header. I'm fine with one of:
* define the constant in this file
* append `// MBEDTLS_SSL_VERIFY_NONE` to this line
* explain the options in a future ttls update
* add an `enum class Authmode` to test_server.h in a future update to hide internal use of mbedtls
* just keep everything as it is now

Choose what you prefer.